x-pack/legacy/plugins/fleet/public/components/agent_health.tsx

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import React from 'react';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { EuiHealth } from '@elastic/eui';
+
+interface Props {
+  agent: any;
+}
+
+export const AgentHealth: React.SFC<Props> = ({ agent }) => {
```

Comment:
this is a simplified component for now as the information returned doesn't contain detailed status information, only `active: (true|false)`

Reply:
Yes, this is due to the events API being added secondary to this. Active has nothing to do with the status though. An unenrolled agent becomes active:false

Reply:
The status needs to be derived from the (events of an agent) + (last checkin with the timeout of the last checkin based on the  agent type, with ephemeral agents having not been seen in X time having no effect to the status , temporary showing in just an nuteral offline state if not seen in polling time from policy * 3 with the default polling time being 30sec, permanent is in a degraded status if not seen in 2x polling time, and an error status of offline if not seen in 4x)

Reply:
Until we have events flowing in, just using the calculation of the last checkin time should be fine